README.md

```diff
@@ -1 +1,44 @@
-# oop-school-library
\ No newline at end of file
+# OPP-SCHOOL-LIBRARY-APP
+
+This is a library app that can help the librarian to record what books are in the library and who borrows them. it will allow him to:
+
+- Add new students or teachers.
+- Add new books.
+- Save records of who borrowed a given book and when.
+
+
+## Built With
+
+- Ruby
+
+## Getting Started
+
+- git clone git@github.com:lilyfrancis/oop-school-library.git
+- cd oop-school-library-app
+## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.